applications/jonny5/src/j5_per_minute.erl

```diff
@@ -161,3 +168,18 @@ metadata(Request) ->
        ,{<<"callee_id_number">>, j5_request:callee_id_number(Request)}
        ,{<<"rate">>, RateObj}
       ]).
+
+-spec calculate_seconds(integer(), integer(), integer(), integer()) -> integer().
+calculate_seconds(_, _, _, 0) ->  0;
+calculate_seconds(NoChargeTime, 0, Minimum, BillingSecs) ->
+    calculate_seconds(NoChargeTime, 60, Minimum, BillingSecs);
+calculate_seconds(NoChargeTime, Increment, Minimum, BillingSecs) ->
+    case BillingSecs =< 0 of
+        'true' -> 0;
+        'false' when BillingSecs =< NoChargeTime ->
+            0;
+        'false' when BillingSecs =< Minimum ->
+            Minimum;
+        'false' ->
+            kz_util:ceiling( BillingSecs / Increment ) * Increment
+    end.
```

Comment:
@onnet hi, this seems wrong to me, or i'm missing something. i think `wht_util:call_cost` may also be wrong.
as an example, if BillingSecs = 23 and NoChargeTime = 10 and Increment = 1, shouldn't the result of `calculate_seconds` be 13 ?